sdk/recoveryservices-siterecovery/Microsoft.Azure.Management.RecoveryServices.SiteRecovery/src/Microsoft.Azure.Management.RecoveryServices.SiteRecovery.csproj

```diff
@@ -6,14 +6,14 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.RecoveryServices.SiteRecovery</PackageId>
     <Description>Provides developers with libraries for the updated recovery services site recovery feature under Azure Resource manager.</Description>
-    <Version>2.1.6-preview</Version>
+    <Version>2.1.7-preview</Version>
```

Comment:
Better to bump version to 2.2.0-preview, since the backend service API version has been updated, see https://semver.org/